packages/link-modules/src/definitions/cart-promotion.ts

```diff
@@ -0,0 +1,55 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+import { LINKS } from "../links"
+
+export const CartPromotion: ModuleJoinerConfig = {
+  serviceName: LINKS.CartPromotion,
+  isLink: true,
+  databaseConfig: {
+    tableName: "cart_promotion",
+    idPrefix: "capromo",
```

Comment:
We might as well just make it `cartpromo` 😄

Reply:
I was joining the prefix convention of both modules. 😄 why not, lets do it.
```suggestion
    idPrefix: "cartpromo",
```